lib/orogen/templates/main.cpp

```diff
@@ -175,7 +174,8 @@ int ORO_main(int argc, char* argv[])
 #endif // OROGEN_SERVICE_DISOCVERY_ACTIVATED
 <% end %>
         ("with-ros", po::value<bool>()->default_value(false), "also publish the task as ROS node, default is false")
-        ("rename", po::value< std::vector<std::string> >(), "rename a task of the deployment: --rename oldname:newname");
+        ("rename", po::value< std::vector<std::string> >(), "rename a task of the deployment: --rename oldname:newname")
+        ("ior-write", po::value<int>(), "the write side of the ior pipe");
```

Comment:
looks like this specifies the file descriptor id used for writing the ior. Maybe it would be better to describe it indeed as a fd.

Reply:
and who is going to specify the ior-write value?

Reply:
> looks like this specifies the file descriptor id used for writing the ior. Maybe it would be better to describe it indeed as a fd.
The ID comes from the command line, trying the id as integer is  simpler.

> and who is going to specify the ior-write value?
The process server spawns the task process, passing the ior write ID. See https://github.com/rock-core/tools-orocosrb/pull/149

Reply:
> The ID comes from the command line, keeping the id as integer is simpler.

I understood @caioaamaral's comment as "add -fd to the command line argument name". At least I would agree with that idea ;-)

Reply:
> "add -fd to the command line argument name"

Ah, that makes sense :laughing:

Reply:
> I understood @caioaamaral's comment as "add -fd to the command line argument name". At least I would agree with that idea ;-)

In the name and/or in its description